hashin.py

```diff
@@ -53,7 +55,7 @@ def _download(url, binary=False):
     return r.read().decode(encoding)
 
 
-def run(spec, file, algorithm, verbose=False):
+def run(spec, file, algorithm, python_versions=[], verbose=False):
```

Comment:
Perhaps it doesn't matter at the moment because this is ever called once. But perhaps the signature will change so you'll be able to do things like `hashin requests pyopenssl flask` then this might call `run` repeatedly and there's a chance that `python_versions` defaults to a non-empty list.